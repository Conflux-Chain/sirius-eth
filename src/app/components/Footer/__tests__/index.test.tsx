import React from 'react';
import { render } from '@testing-library/react';

import { Footer } from '..';

describe('<Footer  />', () => {
  it('should match snapshot', () => {
    const left = [<span key="1">left</span>];
    const rightTop = [<span key="1">rightTop</span>];
    const rightBottom = [
      <span key="1">@2020 Conflux. All Rights Reserved</span>,
    ];
    const loadingIndicator = render(
      <Footer left={left} rightTop={rightTop} rightBottom={rightBottom} />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
