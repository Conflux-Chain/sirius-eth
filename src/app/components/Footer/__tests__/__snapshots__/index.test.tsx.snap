// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer  /> should match snapshot 1`] = `
.c3 {
  -webkit-box-flex: 2;
  -webkit-flex-grow: 2;
  -ms-flex-positive: 2;
  flex-grow: 2;
}

.c2 {
  margin-right: 2.93rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 {
  box-sizing: border-box;
  background-color: #e0e3f2;
  width: 100vw;
  max-width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  padding: 2.2857rem;
}

.c0 .footer-bottom-divider.divider {
  background-color: #d1d5ea;
  margin-top: 1.07rem;
  margin-bottom: 0.57rem;
}

.c1 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  margin: 0 auto;
  max-width: 1368px;
}

@media (max-width:600px) {
  .c3 {
    padding: 0;
  }
}

@media (max-width:600px) {
  .c4 {
    padding: 0 0.21rem;
  }
}

@media (max-width:600px) {
  .c5 {
    padding: 0 0.21rem;
  }
}

@media (max-width:1024px) {
  .c2 {
    display: none;
  }
}

@media (max-width:600px) {
  .c0 {
    padding: 1.14rem 0.93rem;
  }

  .c0 .footer-bottom-divider.divider {
    margin-top: 0.79rem;
    margin-bottom: 0.29rem;
  }
}

<footer
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="c2"
    >
      <span>
        left
      </span>
    </div>
    <div
      class="c3"
    >
      <div
        class="c4"
      >
        <span>
          rightTop
        </span>
      </div>
      <div
        class="jsx-847016443 divider footer-bottom-divider"
        role="separator"
      />
      <div
        class="c5"
      >
        <span>
          @2020 Conflux. All Rights Reserved
        </span>
      </div>
    </div>
  </div>
</footer>
`;
